client/src/components/AdminMessageOptions.js

```diff
@@ -19,24 +22,46 @@ export default function AdminMessageOptions(props) {
   }
 
   function yesDelete() {
-    AnnouncementApi.deleteAnnouncement(props.id)
+    AnnouncementApi.deleteAnnouncement(props.id);
     props.handleRefresh();
   }
 
+  function buttonHandler() {
+    props.setValue(true);
+  }
   return (
-    <><div className="absolute top-0 right-0 translate-y-1/2 -translate-x-1/2 drop-shadow-md px-2 py-0.5 bg-white text-gray-500 border-regal-blue border-2 rounded cursor-pointer">
-      <span className="m-1"><button className="cursor-pointer"><FontAwesomeIcon icon={faComment} size="lg" color="#162750" /></button></span>
-      <span className="m-1"><button className={state ? "hidden" : "cursor-pointer"} onClick={() => handleEdit(props.id)}><FontAwesomeIcon icon={faPenToSquare} size="lg" color="#162750" /></button></span>
-      <span className="m-1"><button className="cursor-pointer" onClick={() => handleDelete()}><FontAwesomeIcon icon={faTrashAlt} size="lg" color="#162750" /></button></span>
-    </div><div>
-        {showModal &&
+    <>
+      <div className="absolute top-0 right-0 translate-y-1/2 -translate-x-1/2 drop-shadow-md px-2 py-0.5 bg-white text-gray-500 border-regal-blue border-2 rounded cursor-pointer">
+        <span className="m-1">
+          <button onClick={buttonHandler} className="cursor-pointer">
+            <FontAwesomeIcon icon={faComment} size="lg" color="#162750" />
+          </button>
+        </span>
+        <span className="m-1">
+          <button
+            className={state ? "hidden" : "cursor-pointer"}
+            onClick={() => handleEdit(props.id)}
+          >
+            <FontAwesomeIcon icon={faPenToSquare} size="lg" color="#162750" />
+          </button>
+        </span>
+        <span className="m-1">
+          <button className="cursor-pointer" onClick={() => handleDelete()}>
+            <FontAwesomeIcon icon={faTrashAlt} size="lg" color="#162750" />
+          </button>
+        </span>
+      </div>
+      <div>
+        {showModal && (
           <DeleteModal
-            message={'Are you sure you want to remove this message?'}
-            buttonConfirmText={'Yes'}
-            buttonCancelText={'No'}
+            message={"Are you sure you want to remove this message?"}
```

Comment:
No need to wrap `{}` since it is not a dynamic data
```suggestion
            message="Are you sure you want to remove this message?"
```